appveyor.yml

```diff
@@ -15,8 +10,6 @@ environment:
     - nodejs_version: "6"
 
 install:
-  - npm -g install npm@latest
```

Comment:
Are we going to run into weird test runs on old node versions with this removed?